dali/operators/reader/numpy_reader_op.cc

```diff
@@ -122,11 +122,6 @@ This argument is mutually exclusive with ``files``.)", nullptr)
 If ``file_root`` is provided, the paths are treated as being relative to it.
 
 This argument is mutually exclusive with ``file_list``.)", nullptr)
-  .AddOptionalArg("register_buffers",
```

Comment:
It will break the code. Please keep the arg and replace the description with:
```
.. warning::
  This argument is currently disabled and left for the backward compatibility. It will be reenabled in the future releases.

Applies **only** to the ``gpu`` backend type.

If true, the device I/O buffers will be registered with cuFile. It is not recommended if the sample
sizes vary a lot.
```

Reply:
ok, will do. Internally I still set it to false no matter what, right?

Reply:
Definitely. We just don't want to break any existing code.
Normally we have an argument deprecation way of doing things, but this time we do not plan to remove it, we just disable it functionality for some time.